### PR TITLE
DOC-6241 Style guide work

### PR DESCRIPTION
**Pull request by paperclypse**
I've updated and reviewed the first batch of Style guide docs.

See https://newrelic.atlassian.net/browse/DOC-6320 for more context.

This spreadsheet describes the work done: https://docs.google.com/spreadsheets/d/1DTuBuzb9HdytSvlFNXcefNFJ5K7d4nfR6IGKTcEi9zE/edit#gid=0

### CODE REVIEW

File: src/new-relic-only/basic-style-guide/style-guide-quick-reference/callouts.mdx [hunk 1]
```diff
@@ -9,23 +9,23 @@ topics:
 japaneseVersion: ''
 ---
 
-NR ONLY
+Callouts direct your attention to information of special importance or to information that doesn't fit smoothly into the main text. Consider your reader. If your doc has too many callouts, it might confuse your reader about what  matters the most and distract them from what does.
```
Comment (urbiz-nr) [inline src/new-relic-only/basic-style-guide/style-guide-quick-reference/callouts.mdx:12]: are there two spaces between what and matters?

**Review by urbiz-nr — COMMENTED** [review]
Looks good!

**Comment by paperclypse** [comment]
@urbiz-nr Looks like you'll need to approve these changes before I can merge them in. :)

**Review by urbiz-nr — APPROVED** [review]
Looks good!

**Review by bradleycamacho — APPROVED** [review]
Looks good to me!

**Pull request merged**